src/main/java/seedu/address/commons/util/FileUtil.java

```diff
@@ -76,8 +78,19 @@ public static String readFromFile(Path file) throws IOException {
      * Writes given string to a file.
      * Will create the file if it does not exist yet.
      */
-    public static void writeToFile(Path file, String content) throws IOException {
-        Files.write(file, content.getBytes(CHARSET));
+    public static void writeToFile(Path file, String content, boolean isReverseOrder) throws IOException {
+        if (isReverseOrder) {
+            List<String> lines = Files.readAllLines(file, Charset.forName(CHARSET));
+            if (lines.size() >= 100) {
+                lines.remove(lines.size() - 1);
+            }
+            lines.add(0, content);
```

Comment:
Food for thought: If `ArrayDeque<String>` is used, if `size >= 100` then `pop back`(to remove back command), then `push front` (to add command string)